pkgs/applications/video/obs-studio/default.nix

```diff
@@ -59,7 +61,7 @@ mkDerivation rec {
     ./Enable-file-access-and-universal-access-for-file-URL.patch
 
     # Lets obs-browser build against CEF 91.1.0+
-    ./Change-product_version-to-user_agent_product.patch
+    # ./Change-product_version-to-user_agent_product.patch
```

Comment:
Once it runs, you should remove those lines and the file from nixpkgs, as the patch was incorporated upstream

Reply:
It does run. Forgot about removing this patch. Fixed. :)